docs/courses/skill.md

```diff
@@ -0,0 +1,192 @@
+# LibreLingo Documentation on Skill
+
+A **Skill** resides below a module within the course structure. It's the lowest element on the cource hierarchy. It contains data regarding how LibreLingo will ask the questions to the users.
+
+A skill is usually focused on a particular topic on which the questions are asked.
```

Comment:
```suggestion
A skill usually is centered around a specific theme. This could be introducing a new grammar concept or just vocabulary to talk about a certain topic.
```